src/java.base/share/classes/java/util/LinkedHashMap.java

```diff
@@ -788,5 +788,21 @@ public final void remove() {
         public final Map.Entry<K,V> next() { return nextNode(); }
     }
 
+    /**
+     * Creates a new, empty LinkedHashMap suitable for the expected number of mappings.
```

Comment:
Adjust wording here to read,

    Creates a new, empty, insertion-ordered LinkedHashMap suitable...

I've used this wording in the CSR.

Reply:
@stuart-marks done.